device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -331,270 +315,10 @@ public void completeFileUpload(FileUploadCompletionNotification notification) th
-     * <a href="https://docs.microsoft.com/en-us/rest/api/iothub/service/twin/updatedevicetwin">PATCH</a>
-     *
-     * @param deviceTwinStatusCallback the IotHubEventCallback callback for providing the status of Device Twin operations. Cannot be {@code null}.
-     * @param deviceTwinStatusCallbackContext the context to be passed to the status callback. Can be {@code null}.
-     * @param genericPropertiesCallBack the TwinPropertyCallBack callback for providing any changes in desired properties. Cannot be {@code null}.
-     * @param genericPropertyCallBackContext the context to be passed to the property callback. Can be {@code null}.
-     *
-     * @throws IllegalArgumentException if the callback is {@code null}
-     * @throws UnsupportedOperationException if called more than once on the same device
-     * @throws IOException if called when client is not opened
-     */
-    public void startDeviceTwin(IotHubEventCallback deviceTwinStatusCallback, Object deviceTwinStatusCallbackContext,
-                                TwinPropertiesCallback genericPropertiesCallBack, Object genericPropertyCallBackContext)
-            throws IOException, IllegalArgumentException, UnsupportedOperationException
-    {
-        this.startTwinInternal(deviceTwinStatusCallback, deviceTwinStatusCallbackContext, genericPropertiesCallBack, genericPropertyCallBackContext);
-    }
-
-    /**
-     * Subscribes to device methods
-     *
-     * @param deviceMethodCallback Callback on which device methods shall be invoked. Cannot be {@code null}.
-     * @param deviceMethodCallbackContext Context for device method callback. Can be {@code null}.
-     * @param deviceMethodStatusCallback Callback for providing IotHub status for device methods. Cannot be {@code null}.
-     * @param deviceMethodStatusCallbackContext Context for device method status callback. Can be {@code null}.
-     *
-     * @throws IOException if called when client is not opened.
-     * @throws IllegalArgumentException if either callback are null.
-     */
-    public void subscribeToDeviceMethod(DeviceMethodCallback deviceMethodCallback, Object deviceMethodCallbackContext,
-                                        IotHubEventCallback deviceMethodStatusCallback, Object deviceMethodStatusCallbackContext)
-            throws IOException, IllegalArgumentException
-    {
-        this.subscribeToMethodsInternal(deviceMethodCallback, deviceMethodCallbackContext, deviceMethodStatusCallback, deviceMethodStatusCallbackContext);
-    }
-
     // Used by multiplexing clients to signal to this client what kind of multiplexing client is using this device client
-    @SuppressWarnings("SameParameterValue") // The connection type is currently only set to "multiplexing client", but it can be set to the deprecated transport client as well.
-    void setConnectionType(IoTHubConnectionType connectionType)
```

Comment:
This method is replaced by ```markAsMultiplexed()``` since it was only ever called with IoTHubConnectionType.Multiplexed and never set back to SINGLE_CLIENT.